app/models/spree/variant_decorator.rb

```diff
@@ -55,6 +55,12 @@ def google_base_item_group_id
       product.slug
     end
 
+    def google_base_item_color 
+      if option_values
+        option_values.first.name if option_values.first
```

Comment:
I tried that.. but it still got the nil class error... I guess some of them don't have a first if it's an empty array? Not sure why in the db there would be but in dev it's calling no method error for option_values.first.name